### DISCUSSION TITLE
gforth exits with an error

### DISCUSSION THREAD

**Discussion by Liorst4** [termux/termux-packages#9592] [Q&A]
After I `pkg install gforth` I tried running it and got this message:
```
gforth: image gforth.fi doesn't seem to be a Gforth (>=0.6) image.
```

I'm running on aarch64.
Does this happen to anyone else?
Any ideas for a fix?

**Comment by 1bildung** [comment]
Yes, same for me in Termux on Android 10 (LineageOS). It seems the rights are not to set correct in $PREFIX/share/gforth and/or it's not the correct image. Quick and dirty fix: copying an image from gforth on my desktop linux (/usr/share/gforth/kernl64l.fi) and start gforth in termux pointing to it:
`gforth -i kernl64l.fi`
